docker/docker-compose.ltc.yml

```diff
@@ -0,0 +1,46 @@
+version: '2'
+services:
+    # ltc is an image of litecoin node which used as base image for ltcd and
+    # ltcctl. The environment variables default values determined on stage of
+    # container start within starting script.
+    ltcd:
+      image: ltcd
+      container_name: ltcd
+      build:
+        context: ltcd/
+      volumes:
+        - shared:/rpc
+        - litecoin:/data
+      environment:
+        - RPCUSER
+        - RPCPASS
+        - NETWORK
+        - DEBUG
+        - MINING_ADDRESS
+      entrypoint: ["./start-ltcd.sh"]
+
+    lnd:
+        image: lnd
+        build:
+          context: ../
+          dockerfile: docker/lnd/Dockerfile
+        environment:
+          - RPCUSER
+          - RPCPASS
+          - NETWORK
+          - CHAIN
+          - DEBUG
+        volumes:
+            - shared:/rpc
+        entrypoint: ["./start-lnd.sh"]
```

Comment:
We need the `links` part here as well, otherwise lnd won't be able to connect to ltcd. The start script of lnd (`docker/lnd/start-lnd.sh`) connects to the host `blockchain`, which we need to create an alias for with the `links` entry.